osu.Game/Graphics/UserInterface/OsuCheckbox.cs

```diff
@@ -33,7 +32,6 @@ public Bindable<bool> Bindable
 
         public string LabelText
         {
-            get => labelSpriteText?.Text;
```

Comment:
Any reason for removing the getter?

Reply:
unsupported by `FillFlow` (and unused anywhere).